src/Data/List/Relation/Unary/All/Properties.agda

```diff
@@ -388,6 +389,18 @@ mapMaybe⁺ {xs = x ∷ xs} {f = f} (px ∷ pxs) with f x
 ... | just v with px
 ...   | just pv = pv ∷ mapMaybe⁺ pxs
 
+------------------------------------------------------------------------
+-- catMaybes
+
+catMaybesAll⁺ : All (Maybe.All P) xs → All P (catMaybes xs)
```

Comment:
Could be proved in a single line from `catMaybesAny⁺` and a call to `All.map` that proves `Maybe.All` implies `Maybe.Any`?

Reply:
> ̀Maybe.All` implies `Maybe.Any`

The implication is the other way around.

Reply:
Yes, and I proved it.